static/js/reactions.js

```diff
@@ -377,11 +372,9 @@ export function remove_reaction(event) {
 
     view.remove_reaction({
         message_id,
-        reaction_type,
-        emoji_name,
-        emoji_code,
         user_list: r.user_ids,
         user_id,
+        ..._.pick(event, emoji_params),
     });
```

Comment:
This last one seems like it'd be better picking the `user_id` and `message_id` fields too.